enterprise/backend/src/metabase_enterprise/serialization/cmd.clj

```diff
@@ -105,9 +106,26 @@
    opts are passed to load-metabase"
   [path :- :string
    opts :- [:map
-            [:abort-on-error {:optional true} [:maybe :boolean]]
             [:backfill? {:optional true} [:maybe :boolean]]]]
-  (v2-load-internal! path opts :token-check? true))
+  (let [start    (System/currentTimeMillis)
+        err      (atom nil)
+        report   (try
+                   (v2-load-internal! path opts :token-check? true)
+                   (catch Exception e
+                     (reset! err e)))
+        imported (into (sorted-set) (map (comp :model last)) (:seen report))]
```

Comment:
Ah! This is where you made the update we discussed elsewhere. It would be nice if updated the other spot too.